frontend/src/hooks/useRole.ts

```diff
@@ -0,0 +1,5 @@
+import { useGetSessionQuery } from "@store/services/api";
+export const useRole = () => {
+  const { status } = useGetSessionQuery();
```

Comment:
This will return query promise status (fulfilled, rejected), not the role. 

This is a screenshot from getSession query.
![image](https://github.com/uaem-na/donations-tracker/assets/8529473/85a3228c-141c-47b2-a323-7b2b2e22f0b8)